bnr-utils/nz_azConnector/nz_azConnector.go

```diff
@@ -384,7 +384,8 @@ func main() {
                         if r.err != nil {
                             // stopping right here so that we
                             // don't keep on uploading when one has failed
-                            log.Fatalf("%s: %v", r.job, r.err)
+                            log.Println("Error while uploading file. Ensure azure storage account name, azure key and container name are correct. If error persists contact IBM support team.")

```

Comment:
@shabbir10july lets make sure our uploadJob and downloadJob can be printed correctly.

```go
// at the top of the file

func (c Conn) String() string {
    return fmt.Sprintf("account:%s container:%s", c.azaccount, c.azcontainer)
}

func (j Job) String() string {
    return fmt.Sprintf("conn:[%s] backupDir:%s id:%s", j.conn, j.backupdir, j.uniqueid)
}

func (u uploadJob) String() string {
    return fmt.Sprintf("%s file:%s", u.job, u.absfilepath)
}

func (d downloadJob) String() string {
    return fmt.Sprintf("conn:[%s] blob:%s file:%s", d.conn, d.blobname, d.outfilepath)
}
```

With that change these two lines to

```go
log.Fatalf("Error uploading file. Ensure azure storage account name, azure key and container name are correct. If error persists contact IBM support team: %s`,
    *r.job)
```

Reply:
okay done